dev/preview/distribute-images.sh

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+[[ "$(kubectx -c)" == "harvester" ]] || ( echo "Set kubectx to 'harvester'."; exit 1)
+
+while getopts i:s: flag
+do
+    case "${flag}" in
+        i) IMAGEID="${OPTARG}";;
+        s) STORAGECLASS="${OPTARG}";;
+        *) ;;
+    esac
+done
+
+NODES=$(kubectl get nodes -o=jsonpath='{.items[*].metadata.name}')
+NAMESPACE="distribute-${IMAGEID}"
```

Comment:
```suggestion

// We don't delete the namespace "distribute-${IMAGEID} because we want to avoid
// images from being garbage collected
NODES=$(kubectl get nodes -o=jsonpath='{.items[*].metadata.name}')
NAMESPACE="distribute-${IMAGEID}"
```
```suggestion

NODES=$(kubectl get nodes -o=jsonpath='{.items[*].metadata.name}')
NAMESPACE="distribute-${IMAGEID}"
```